rsconnect/api.py

```diff
@@ -981,6 +991,21 @@ def make_deployment_name(self, title, force_unique):
 
         return name
 
+    def list_runtime_caches(self):
+        return self.client.system_caches_runtime_list()
```

Comment:
```suggestion
    @property
    def runtime_caches(self):
        return self.client.system_caches_runtime_list()
```
The design pattern for Connect Executor is to make chainable functions, and to make non-chainable information into properties.